bigtable-hbase-1.x-parent/bigtable-hbase-1.x-benchmarks/clirr-ignored-differences.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- see http://www.mojohaus.org/clirr-maven-plugin/examples/ignored-differences.html -->
+<differences>
+  <difference>
+    <differenceType>8001</differenceType>
+    <className>**</className>
```

Comment:
I'm surprised this is necessary for these changes. In any case, ** is too greedy. Can this be more specific?

Reply:
This is an internal artifact not published to customers. CLIRR should be disabled altogether